tap_kafka/serialization/json_with_no_schema.py

```diff
@@ -0,0 +1,32 @@
+import orjson
+from confluent_kafka.serialization import Deserializer
+from confluent_kafka.serialization import SerializationError
+
+
+class JSONSimpleDeserializer(Deserializer):
+    """
+    Deserializes a Python object from JSON formatted bytes.
+    """
+
+    def __init__(self):
+        pass
```

Comment:
I reckon this is not needed to have here

Reply:
thanks, fixed by c114942a780a3d39e7f1f54043ea0bf3ac193ca3